dist/blockstack.js

```diff
@@ -307,6 +308,39 @@ function signUserOut() {
   return userSession.signUserOut();
 }
 
+/**
+ * Generates an authentication request that can be sent to the Blockstack
+ * browser for the user to approve sign in. This authentication request can
+ * then be used for sign in by passing it to the `redirectToSignInWithAuthRequest`
+ * method.
+ *
+ * *Note: This method should only be used if you want to roll your own authentication
+ * flow. Typically you'd use `redirectToSignIn` which takes care of this
+ * under the hood.*
+ *
+ * @param  {String} transitPrivateKey - hex encoded transit private key
+ * @param {String} redirectURI - location to redirect user to after sign in approval
+ * @param {String} manifestURI - location of this app's manifest file
+ * @param {Array<String>} scopes - the permissions this app is requesting
+ * @param {String} appDomain - the origin of this app
+ * @param {Number} expiresAt - the time at which this request is no longer valid
+ * @param {Object} extraParams - Any extra parameters you'd like to pass to the authenticator.
+ * Use this to pass options that aren't part of the Blockstack auth spec, but might be supported
+ * by special authenticators.
+ * @return {String} the authentication request
+ */
+function makeAuthRequest(transitPrivateKey, redirectURI, manifestURI, scopes) {
+  var appDomain = arguments.length > 4 && arguments[4] !== undefined ? arguments[4] : window.location.origin;
+  var expiresAt = arguments[5];
+  var extraParams = arguments.length > 6 && arguments[6] !== undefined ? arguments[6] : {};
+
+  console.warn('DEPRECATION WARNING: The makeAuthRequest() function will be deprecated in the ' + 'next major release of blockstack.js. Use UserSession to configure your auth request.');
```

Comment:
Do we want to have JSDoc `@deprecated` on these as well? They won't show up in editors until we make some improvements to the build / npm packaging, but I think they will show up in the generated docs?

Reply:
I had them initially but for some reason documentation.js simply ignores the `@deprecated` tags.